sampling/sampling.go

```diff
@@ -34,6 +34,8 @@ var (
 //
 // The returned model.BatchProcessor does not guarantee order preservation
 // of events retained in the batch.
+//
+// TODO(axw) only discard non-RUM unsampled transactions.
```

Comment:
Follow up issue for this?

Reply:
Yep, there's already an issue: https://github.com/elastic/apm-server/issues/6260